### PR TITLE
Helm3.1 onap sync with cn split repos

### PR DESCRIPTION
**Pull request by lionelgo**
Distribution compat: RHEL8.3

### CODE REVIEW

File: openshift/oai-mme-build-config.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
     source:
         git:
             uri: "https://github.com/OPENAIRINTERFACE/openair-mme.git"
-            ref: "develop"
+            ref: "helm3.1-onap-sync-with-cn-split-repos"
```
Comment (rdefosse) [inline openshift/oai-mme-build-config.yml:32]: Please do not change the reference branch --> keep `develop`.

It should be a local change on your local workspace. not to be pushed.

**Review by rdefosse — CHANGES_REQUESTED** [review]

File: etc/mme_fd.sprint.conf [hunk 2]
```diff
@@ -166,4 +166,4 @@ LoadExtension = "/usr/local/lib/freeDiameter/dict_S6c.fdx";
 # incoming connections from other peers, see the acl_wl.fdx? extension which 
 # allows exactly this. 
 
-ConnectPeer= "@HSS_FQDN@" { ConnectTo = "@HSS_IP_ADDR@"; No_SCTP ; No_IPv6; Prefer_TCP; No_TLS; port = 3868;  realm = "@REALM@";};
```
Comment (rdefosse) [inline etc/mme_fd.sprint.conf:169]: Just for my knowledge. What is the impact of removing the realm in this line?

Comment (lionelgo) [reply etc/mme_fd.sprint.conf:169]: HSS and MME can have different realms, this is what happens actually on OC. This REALM is actually the MME realm.

Comment (rdefosse) [reply etc/mme_fd.sprint.conf:169]: Thx

**Review by rdefosse — APPROVED** [review]

**Comment by rdefosse** [comment]
@lionelgo 

DsTester failed.

File: src/mme_app/mme_config.h [hunk 3]
```diff
@@ -75,6 +75,7 @@
 #define MME_CONFIG_STRING_S6A_CONFIG "S6A"
 #define MME_CONFIG_STRING_S6A_CONF_FILE_PATH "S6A_CONF"
 #define MME_CONFIG_STRING_S6A_HSS_HOSTNAME "HSS_HOSTNAME"
+#define MME_CONFIG_STRING_S6A_HSS_REALM    "HSS_REAL"
```
Comment (rdefosse) [inline src/mme_app/mme_config.h:78]: @lionelgo typo?

**Review by rdefosse — APPROVED** [review]

File: src/mme_app/mme_config.h (hunk 3 above)
Comment (lionelgo) [reply src/mme_app/mme_config.h:78]: Yes Typo, something else no good, I check...

Comment (rdefosse) [reply src/mme_app/mme_config.h:78]: if CI OK, let's be it...

**Pull request merged**